src/site/content/en/blog/submit-your-proposals-for-interop-2023/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Submit your proposals for Interop 2023"
+subhead:
+    "We are thrilled at the progress of Interop 2022 that we’ve made together and are excited to announce  Interop 2023 is happening!"
+description:
+    "Learn more about how to submit a proposal for Interop 2023"
+authors:
+  - rachelandrew
+  - robertnyman
+  - foolip
+hero: "image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/ZZjlTqQijYN8FocvisWC.jpg"
+alt: "A picture of three post-its and a marker pen"
+date: 2022-09-21
+updated: 2022-09-21
+tags:
+  - blog
+  - css
+---
+
+The Chrome team has worked with other browser vendors and partners in the ecosystem to help improve browser compatibility since 2019, starting with the [MDN Developer Needs Assessment](https://insights.developer.mozilla.org/reports/mdn-web-developer-needs-assessment-2019.html) survey and most recently on [Interop 2022](/interop-2022/). For Interop 2022, all major browser vendors and other stakeholders worked together to improve top browser compatibility issues identified by web developers. We are thrilled at [the progress that we’ve made together](https://wpt.fyi/interop-2022) and are excited to announce Interop 2023 is happening!
+
+<figure>
+  {% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/8ryQRYl1NBObumG4IA97.png", alt="Showing three circles with scores: 71 for Chrome Dev and Edge Dev, 74 for Firefox Nightly and 73 for Safari Technology Preview", width="800", height="375" %}
+  <figcaption>March 2022.</figcaption>
+</figure>
+
+
+
+
+<figure>
+  {% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/8fXDfdtPWAh2xMF6iiYT.png", alt="Showing three circles with scores: 80 for Chrome Dev and Edge Dev, 82 for Firefox Nightly and 86 for Safari Technology Preview", width="800", height="501" %}
+  <figcaption>September 2022.</figcaption>
+</figure>
+
+The Interop 2023 proposal process is now open for submissions. Suggestions can include features that behave differently across browsers, or features that are not yet implemented in all browsers. Keep in mind that proposals with high quality specifications and web platform tests are more likely to be included, as this project is about reaching interoperability rather than specifying new features. [The proposal submission process](https://github.com/web-platform-tests/interop/issues/new?template=focus-area-proposal.yml) ends October 15, 2022
```

Comment:
```suggestion
The Interop 2023 proposal process is now open for submissions. Suggestions can include features that behave differently across browsers, or features that are not yet implemented in all browsers. Keep in mind that proposals with high quality specifications and web platform tests are more likely to be included, as this project is about reaching interoperability rather than specifying new features. [The proposal submission process](https://github.com/web-platform-tests/interop/issues/new?template=focus-area-proposal.yml) ends October 15, 2022.
```